src/status_im2/common/resources.cljs

```diff
@@ -97,5 +130,5 @@
   (get ui k))
 
 (defn get-themed-image
```

Comment:
this method previously needed both keys, to me it seems better to use one key and have the themed options defined in the map. 👍 / 👎 ?